maintenance_equipment_tags/models/maintenance_equipment_tag.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+from random import randint
+
+
+class MaintenanceEquipmentTag(models.Model):
+
+    _name = 'maintenance.equipment.tag'
+    _description = 'Maintenance Equipment Tag'
+
+    def get_default_color_value(self):
+        return randint(1, 15)
+
+    name = fields.Char(string="Equipment Tag", required=True)
+    color = fields.Integer(
+        string='Color Index',
+        default=lambda self: self.get_default_color_value()
+    )
+    equipment_ids = fields.Many2many(
```

Comment:
I don't think this field is necessary here...
In any case, `string='Employees'` is not appropriate.

Reply:
Sorry, copy-pasted from hr and didnt change it. I think it doesnt do any bad and can be useful if someone wants to access equipment from tags